include/drm.h

```diff
@@ -1,8 +1,8 @@
 #ifndef INITSHIM_DRM_H
 #define INITSHIM_DRM_H
 
-#include <libdrm/drm.h>
-#include <libdrm/drm_mode.h>
+#include <drm/drm.h>
+#include <drm/drm_mode.h>
```

Comment:
This will probably cause a build failure in initshim-deploy. If there's an easy way to ifdef building in AOSP tree that'd be preferred. If not, I can change it in -deploy

Reply:
I can add `__has_include` to check the headers path or add `-DBUILDING_IN_AOSP` to `cflags` and add `#ifdef`. Not sure what is better though.

Reply:
_has_include seems cleaner for me.

Reply:
Done.